ironfish-cli/src/commands/wallet/send.ts

```diff
@@ -143,55 +157,116 @@ export class Send extends IronfishCommand {
       this.exit(1)
     }
 
-    if (fee == null) {
-      let suggestedFee = ''
-      try {
-        const response = await client.estimateFee({
-          fromAccountName: from,
-          receives: [
-            {
-              publicAddress: to,
-              amount: CurrencyUtils.encode(amount),
-              memo: memo,
-            },
-          ],
-        })
+    if (expiration !== undefined && expiration < 0) {
+      this.log('Expiration sequence must be non-negative')
+      this.exit(1)
+    }
+
+    let rawTransactionResponse: string
+    if (fee == null && feeRate == null) {
+      const feeRates = await client.estimateFeeRates()
+      const feeRateOptions: { value: number; name: string }[] = []
+
+      const createTransactionRequest: CreateTransactionRequest = {
+        sender: from,
+        receives: [
+          {
+            publicAddress: to,
+            amount: CurrencyUtils.encode(amount),
+            memo,
+            assetId,
+          },
+        ],
+        expiration: expiration,
+      }
+
+      const allPromises = []
+      if (feeRates.content.low) {
+        allPromises.push(
+          client.createTransaction({
+            ...createTransactionRequest,
+            feeRate: feeRates.content.low,
+          }),
+        )
+      }
 
-        switch (flags.priority) {
-          case 'low':
-            suggestedFee = CurrencyUtils.renderIron(response.content.low)
+      if (feeRates.content.medium !== feeRates.content.low) {
+        allPromises.push(
+          client.createTransaction({
+            ...createTransactionRequest,
+            feeRate: feeRates.content.medium,
+          }),
+        )
+      }
+
+      if (
+        feeRates.content.high !== feeRates.content.low &&
+        feeRates.content.high !== feeRates.content.medium
+      ) {
+        allPromises.push(
+          client.createTransaction({
+            ...createTransactionRequest,
+            feeRate: feeRates.content.high,
+          }),
+        )
+      }
+
+      const createResponses = await Promise.all(allPromises)
+      createResponses.forEach((createResponse, index) => {
+        const rawTransactionBytes = Buffer.from(createResponse.content.transaction, 'hex')
+        const rawTransaction = RawTransactionSerde.deserialize(rawTransactionBytes)
+
+        let name
+        switch (index) {
```

Comment:
I thought of that but its not, because Low, Medium, High are really just descriptions user see. So it makes sense to have the first estimate fee as Low.